cypress/integration/doiForm.spec.js

```diff
@@ -97,9 +97,10 @@ describe("DOI form", function () {
     cy.get(".MuiListItem-container", { timeout: 10000 }).should("have.length", 2)
 
     // Go to DOI form
-    cy.get("button[type=button]").contains("Next").click()
-    cy.get("button").contains("Add DOI information (optional)", { timeout: 10000 }).click()
-    cy.get("div[role='dialog']").should("be.visible")
+    cy.openDOIForm()
+    // cy.get("button[type=button]").contains("Next").click()
+    // cy.get("button").contains("Add DOI information (optional)", { timeout: 10000 }).click()
+    // cy.get("div[role='dialog']").should("be.visible")
```

Comment:
hmm if this is moved to `cy.openDOIForm` then maybe we can remove the lines ?

Reply:
commented code is removed.